src/components/shared/Extensions/Accordion/partials/AccordionHeader.tsx

```diff
@@ -7,7 +7,7 @@ const displayName = 'Extensions.Accordion.partials.AccordionHeader';
 const AccordionHeader: FC<AccordionItemProps> = ({ title, isOpen }) => (
   <div>
     <div className="flex justify-between items-center pb-2 pt-4">
-      {title}
+      {title as React.ReactNode}
```

Comment:
You shouldn't use casting here.
Please set 
`title: string | ReactNode` in Props

Change everywhere where you use it.